src/components/modal/components/VoidProduct.tsx

```diff
@@ -286,7 +288,6 @@ export default function VoidProduct({
     try {
       setIsLoading(true);
       let txResponse: TransactionResponse;
-      const isMetaTx = Boolean(coreSdk?.isMetaTxConfigSet && signer);
```

Comment:
does the isMetaTx check if signer is truthy? (if it's necessary, otherwise ignore)